docs/upgrade.html

```diff
@@ -118,6 +118,12 @@
     <li> The new consumer API has been marked stable. </li>
 </ul>
 
+<h5><a id="upgrade_101_notable" href="#upgrade_101_notable">Notable changes in 0.10.0.1</a></h5>
```

Comment:
This change will be in 0.10.1.0, not 0.10.0.1 (the latter only includes bug fixes).